### DISCUSSION TITLE
widget.setProperty adds DATA_SET control custom property

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#232] [Q&A] [answered]
Hi.
What does this mean? Can we manually set data (for example to preview how it looks) or change data_type by setProperty?
@MyoungXUE @silver-zepp

**Comment by silver-zepp** [comment, marked as answer]
It's just an empty object, for you to contain and retrieve your personal data (per object) on when you need it later. Like for example `DATA_SET.my_button_last_position` @zdunex25
